UIMentoringSession/MenteesChallenges/Pets Main Card UI Chellenge/YefimYakunin/PetCollectionCell/YeYa_PetCell.swift

```diff
@@ -0,0 +1,33 @@
+import UIKit
+
+class YeYa_PetCell: UICollectionViewCell {
+    
+    @IBOutlet private weak var shadowView: UIView!
+    @IBOutlet private weak var mainView: UIView!
+    @IBOutlet private weak var petImage: UIImageView!
+    @IBOutlet private weak var nameLabel: UILabel!
+    @IBOutlet private weak var breedLabel: UILabel!
+    @IBOutlet private weak var ageLabel: UILabel!
+    
+    private func setupLayout() {
```

Comment:
move private methods after all other
Order of methods:
1. override methods
2. internal methods
3. private methods

Reply:
@IBOutlet private weak var breedLabel: UILabel!
    @IBOutlet private weak var ageLabel: UILabel!
    
    override func awakeFromNib() {
        super.awakeFromNib()
        setupLayout()
    }

Corrected the order of methods